assets/python/ml/pyml:data_input:train_test_split:sklearn.j2.py

```diff
@@ -0,0 +1,38 @@
+# ----------------------------------------------------------------- #
+#                                                                   #
+#   Workflow Unit to perform a train/test split                     #
```

Comment:
The docstrings like this one are a waste of space, as they just duplicate the information in the unit name. Either add more explanation (preferred) or remove

Reply:
Docstring has been expanded, to describe in more detail what the unit does and how to configure it.